frontend/src/Components/Content/Albums.tsx

```diff
@@ -1,39 +1,48 @@
+import React from "react"
 
 export type Album = {
-  name: string,
-  imgUrl: string,
-  description: string
+    name: string,
+    imgUrl: string,
+    description: string
 }
 
-export default function AlbumComponent({ albums }: { albums: Array<Album> }) {
-  function generateInnerContent(album: Album) {
-    return (
-        <>
-            <div style={AlbumNameAlignStyle}>
-                <div style={AlbumNameStyle} translate="no">{album.name}
+type AlbumProps = {
+    albums: Array<Album>
+}
+
+export const AlbumComponent: React.FC<AlbumProps> = ({ albums }) => {
+
+    // this is former code. Leaving in for documentation.
+    // export default function AlbumComponent({ albums }: AlbumProps) {
+    // export default function AlbumComponent({ albums }: { albums: Array<Album> }) {
+    function generateInnerContent(album: Album) {
+        return (
+            <>
+                <div style={AlbumNameAlignStyle}>
+                    <div style={AlbumNameStyle} translate="no">{album.name}
+                    </div>
+                    <div style={AlbumNameSpacingStyle} translate="no">
+                    </div>
                 </div>
-                <div style={AlbumNameSpacingStyle} translate="no">
```

Comment:
I'm not necessarily against it and I would have to run the code to see the difference, but what is the reason for moving the `AlbumNameSpacingStyle` div inside the `AlbumNameAlignStyle` div here? Was there any difference visually?